addons/arsenal/functions/fnc_onKeyDown.sqf

```diff
@@ -183,6 +178,14 @@ if (!isNull _loadoutsDisplay) then {
                 };
             };
         };
+        if (_updateSearchResults) then {
```

Comment:
the only hangup is here,
`handleSearchBar` will check `ctrlText _control`
if you are typing letters, the letters will show up in that as you expect
if you hit backspace, it apparently doesn't take effect until after this returns

normally I'd just use execNextFrame but this needs to work in 3den
any ideas?

Reply:
Isn't there an EH for CT_EDIT changes?

Reply:
https://community.bistudio.com/wiki/User_Interface_Event_Handlers#onEditChanged 2.14

Reply:
With above I'd say we can just have live search always on.